metrics/api/src/main/java/io/helidon/metrics/api/MetricsConfigBlueprint.java

```diff
@@ -250,6 +251,16 @@ static List<Tag> createTags(String pairs) {
     @Option.Default(BuiltInMeterNameFormat.DEFAULT)
     BuiltInMeterNameFormat builtInMeterNameFormat();
 
+    /**
+     * Default base units for timers if not specified on the timer builder.
+     * If neither the configuration nor the builder specifies the timer units the results can depend on
+     * the underlying provider implementation of timers and, therefore, might not be predictable.
+     *
+     * @return default {@link java.util.concurrent.TimeUnit} to use as the base unit for timers
+     */
+    @Option.Configured("timers.base-units-default")
+    Optional<TimeUnit> timerBaseUnitsDefault();
+
```

Comment:
Why not provide a default (for this default) and dtop the optional here? Seems simpler and predictable.

Reply:
Not backward compatible maybe?

Reply:
Yes, that was pretty much my thinking. The current (before the PR) 4.x behavior is that the Helidon layer delegates to the Micrometer layer to decide what units to apply. Using an `Optional` here allows us to detect when the user config actually specified something so we know whether to use that or delegate to Micrometer.

Reply:
So Micrometer is not always using the same defaults?

Reply:
In typical (default) usage, Micrometer will use Prometheus metric implementations (including timer) but that's not guaranteed. Users could have Micrometer itself use some other implementation underneath, in which case the default behavior might be different. The Helidon code would not necessarily see that because it uses the Micrometer API, and our code does not look behind the scenes to see if the user has plugged in some other underlying implementation under Micrometer.

I'm not aware of any of our users actually doing this, but there's no reason they couldn't and no inherent reason we'd hear about it.